packages/dev/s2-docs/src/SearchMenu.tsx

```diff
@@ -1,21 +1,41 @@
 'use client';
 
 import {ActionButton, Content, Heading, IllustratedMessage, SearchField, Tag, TagGroup} from '@react-spectrum/s2';
-import {Autocomplete, Dialog, Key, OverlayTriggerStateContext, Provider} from 'react-aria-components';
+import {Autocomplete, Dialog, Key, OverlayTriggerStateContext, Provider, Separator as RACSeparator, useFilter} from 'react-aria-components';
 import Close from '@react-spectrum/s2/icons/Close';
 import {ComponentCardView} from './ComponentCardView';
 import {getLibraryFromPage, getLibraryFromUrl} from './library';
+import {iconAliases} from './iconAliases.js';
+import {iconList, IconSearchSkeleton} from './IconSearchView';
```

Comment:
I think you need to move IconSearchSkeleton into a separate file otherwise the lazy import below won't do anything (it already got bundled in synchronously).